packages/amazonq/test/e2e/amazonq/framework/messenger.ts

```diff
@@ -153,17 +153,24 @@ export class Messenger {
         return this.getActionsByFilePath(filePath).some((action) => action.name === actionName)
     }
 
+    async waitForText(text: string, waitOverrides?: MessengerOptions) {
+        await this.waitForEvent(() => {
+            console.log(this.getChatItems())
```

Comment:
remove this

Reply:
tbh i think the console logs are useful. When you have a set of e2e tests that take a look time to run its usually a lot easier to see why they failed by looking at what chat items are present

Reply:
We should technically only log chat items when we encounter an error'd test though, otherwise theres just going to be a lot of spam in the logs